src/nft/components/card/containers.tsx

```diff
@@ -0,0 +1,849 @@
+
+  if (noContent) {
+    return <NoContentContainer height={getHeightFromAspectRatio(uniformAspectRatio, renderedHeight)} />
+  }
+
+  return (
+    <StyledMediaContainer>
+      <StyledImage
+        src={src}
+        $aspectRatio={getMediaAspectRatio(uniformAspectRatio, setUniformAspectRatio)}
+        imageLoading={!loaded}
+        draggable={false}
+        onError={() => setNoContent(true)}
+        onLoad={(e) => {
+          handleUniformAspectRatio(uniformAspectRatio, e, setUniformAspectRatio, renderedHeight, setRenderedHeight)
+          setLoaded(true)
+        }}
+      />
+    </StyledMediaContainer>
+  )
+}
+
+interface MediaProps {
+  mediaSrc?: string
+  tokenId?: string
+  shouldPlay: boolean
+  setCurrentTokenPlayingMedia: (tokenId: string | undefined) => void
+}
+
+const PlaybackButton = styled.div<{ pauseButton?: boolean }>`
+  display: ${({ pauseButton }) => (pauseButton ? 'block' : 'none')};
+  color: ${({ theme }) => theme.accentTextLightPrimary};
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  z-index: 1;
+  margin-left: calc(100% - 50px);
+  transform: translateY(-56px);
+
+  @media screen and (max-width: ${BREAKPOINTS.sm}px) {
+    display: block;
+  }
+
+  ${StyledImageContainer}:hover & {
+    display: block;
+  }
+`
+
+const StyledVideo = styled.video<{
+  $aspectRatio?: string
+}>`
+  width: 100%;
+  aspect-ratio: ${({ $aspectRatio }) => $aspectRatio};
+`
+
+const StyledInnerMediaContainer = styled(Row)`
+  position: absolute;
+  left: 0px;
+  top: 0px;
+`
+
+const Video = ({
+  src,
+  mediaSrc,
+  tokenId,
+  uniformAspectRatio = UniformAspectRatios.square,
+  setUniformAspectRatio,
+  renderedHeight,
+  setRenderedHeight,
+  shouldPlay,
+  setCurrentTokenPlayingMedia,
+}: MediaProps & ImageProps) => {
+  const vidRef = useRef<HTMLVideoElement>(null)
+  const [noContent, setNoContent] = useState(!src)
+  const [imageLoaded, setImageLoaded] = useState(false)
+
+  useEffect(() => {
+    if (shouldPlay && vidRef.current) {
+      vidRef.current.play()
+    } else if (!shouldPlay && vidRef.current) {
+      vidRef.current.pause()
+    }
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [shouldPlay, vidRef.current])
+
+  if (noContent) {
+    return <NoContentContainer height={getHeightFromAspectRatio(uniformAspectRatio, renderedHeight)} />
+  }
+
+  return (
+    <>
+      <StyledMediaContainer>
+        <StyledImage
+          src={src}
+          $aspectRatio={getMediaAspectRatio(uniformAspectRatio, setUniformAspectRatio)}
+          imageLoading={!imageLoaded}
+          draggable={false}
+          onError={() => setNoContent(true)}
+          onLoad={(e) => {
+            handleUniformAspectRatio(uniformAspectRatio, e, setUniformAspectRatio, renderedHeight, setRenderedHeight)
+            setImageLoaded(true)
+          }}
+          $hidden={shouldPlay}
+        />
+      </StyledMediaContainer>
+      {shouldPlay ? (
+        <>
+          <PlaybackButton pauseButton={true}>
```

Comment:
While hovering, the play button is very close to the top of the container
<img width="343" alt="Screen Shot 2023-03-13 at 10 50 05 " src="https://user-images.githubusercontent.com/11512321/224794362-57697cdb-1fd2-4c9f-85e0-1a5771a9ab3b.png">